src/main/java/org/gridsuite/useradmin/server/service/ScheduledAnnouncementCleaner.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.useradmin.server.service;
+
+import org.gridsuite.useradmin.server.entity.AnnouncementEntity;
+import org.gridsuite.useradmin.server.repository.AnnouncementRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.List;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+@Service
+public class ScheduledAnnouncementCleaner {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ScheduledAnnouncementCleaner.class);
+
+    private final AnnouncementRepository announcementRepository;
+
+    public ScheduledAnnouncementCleaner(AnnouncementRepository announcementRepository) {
+        this.announcementRepository = announcementRepository;
+    }
+
+    @Scheduled(cron = "${clean-announcement-cron}", zone = "UTC")
+    public void deleteExpiredAnnouncements() {
+        Instant now = Instant.now();
+        LOGGER.info("delete expired announcement cron starting execution at {}", now);
```

Comment:
No need to add the time, logback will add it in its format.
```suggestion
        LOGGER.info("Starting Delete expired announcement cron job");
```